Features/RoleUI/RoleUI.feature

```diff
@@ -0,0 +1,166 @@
+    Given I am on the Roles page
+    And I create a new role
+    And I set "私は日本の名前で試してみる必要が" as the "Name"
+    When I click on "Save" button
+    Then the Role is successfully published
+
+  @javascript @edge
+  Scenario: creating a Role with an empty name triggers an obligation to fill the name
+    Given I am on the Roles page
+    And I create a new role
+    And I set empty as the "Name"
+    When I click on "Save" button
+    Then I see a message asking for the field "Name" to be filled
+    And the Role is not published
+
+  @javascript @edge
+  Scenario: creating a Role with an existing identifier fails validation
+    Given I have an "Organizers" role
+    And I create a new role
+    And I set "Organizers" as the "Name"
+    When I click on "Save" button
+    Then I see a message saying that the name already exists
+    And the Role is not published
+
+  @javascript @common
+  Scenario: cancel the creation of a role
+    Given I am on the Roles page
+    And I create a new role
+    And I set "Organizers" as the "Name"
+    When I click on "Cancel" button
+    Then the Role is not published
+
+  ##
+  #
+  # Updating Roles
+  #
+  ##
+  @javascript @common
+  Scenario: Validate the existence of expected fields when editing a role
+    Given I have an "Organizers" role
+    When I edit the "Organizers" role
+    Then I should see an "Edit <Organizers>" label
+    And I should see a "Name" field
+    And I should see a "Save" button
+    And I should see a "Cancel" button
+
+  @javascript @common
+  Scenario: Update a role
+    Given I have an "Organizers" role
+    And I edit the "Organizers" role
+    And I set "old_Organizers" as the "Name"
+    When I click on "Save" button
+    Then the Role is successfully published
+
+  @javascript @edge
+  Scenario: Updating a Role with an empty name triggers a message asking to fill the name
+    Given I have an "Organizers" role
+    And I edit the "Organizers" role
+    And I set empty as the "Name"
+    When I click on "Save" button
+    Then I see a message asking for the field "Name" to be filled
+    And the Role is not published
+
+  @javascript @edge
+  Scenario: Updating a Role with an existing identifier fails validation
+    Given I have an "Organizers" role
+    And I have a "Security" role
+    And I edit the "Organizers" role
+    And I set "Security" as the "Name"
+    When I click on "Save" button
+    Then I see a message saying that the name already exists
+    And the Role is not published
+
+  @javascript @common
+  Scenario: cancel the update of a role
+    Given I have an "Organizers" role
+    And I edit the "Organizers" role
+    And I set "Security" as the "Name"
+    When I click on "Cancel" button
+    Then the Role is not published
+
+  ##
+  #
+  # Deleting Roles
+  #
+  ##
+  @javascript @common
+  Scenario: Delete a role
+    Given I have an "Organizers" role
+    When I delete the "Organizers" role
+    Then the Role is successfully deleted
```

Comment:
same here as published, could just as well be a sentence verifying it is no longer present in list of roles, would be more clear as to how you expect the change to affect the application.